weechat-android/proguard.txt

```diff
@@ -22,9 +22,13 @@
 # strip all logger calls
 # i hope the if (DEBUG) checks will get stripped too
 -assumenosideeffects class org.slf4j.Logger {
-    public void error(...);
-    public void warn(...);
-    public void info(...);
+    # public void error(...);
```

Comment:
Just remove these if they're not needed; git will have the revision history if we need it back.